docs/en/setup/backend/aws-firehose-receiver.md

```diff
@@ -18,3 +18,10 @@ CloudWatch metrics with S3 -->  CloudWatch Metric Stream (OpenTelemetry formart)
 
 1. Only OpenTelemetry format is supported (refer to [Metric streams output formats](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch-metric-streams-formats.html))
 2. Only HTTPS could be accepted, you could directly enable TLS and set the receiver to listen 443, or put the receiver behind a gateway with HTTPS (refer to [Amazon Kinesis Data Firehose Delivery Stream HTTP Endpoint Delivery Specifications](https://docs.aws.amazon.com/firehose/latest/dev/httpdeliveryrequestresponse.html))
+
+The OTEL proto in this module copy from [opentelemetry-proto v0.7.0](https://github.com/open-telemetry/opentelemetry-proto/tree/v0.7.0) as AWS firehose only support OTEL 0.7, and rewrite follows :
+
+1. Only remain metrics part
```

Comment:
What do you mean only metrics? Do you mean tracing and logs excluded?

Reply:
I only copied metrics part, because aws-firehose-receiver only processes metrics data.
BTW I have updated the description of this PR and will revert the doc later.